dataactvalidator/config/sqlrules/a10_appropriations.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+    approp.row_number,
+    approp.borrowing_authority_amount_cpe,
+    SUM(sf.amount) as total_amount
```

Comment:
@nmonga91 I don't think this was widely communicated, but we have an informal naming convention going for aggregate column aliases: `[something description]_[aggregate_being_applied].

So in this case, `SUM(sf.amount)` might be aliased as `sf_133_sum` or even `sf_133_1340_1440_sum`, though I see how that could get cumbersome in rules that involve a lot of SF 133 lines.